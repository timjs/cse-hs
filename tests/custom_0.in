3
CAPS(A,A)
a(a,a)
boring
