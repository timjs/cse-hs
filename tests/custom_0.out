CAPS(A,2)
a(a,2)
boring
